specification/cognitiveservices/data-plane/LUIS/Runtime/preview/v3.0/LUIS-Runtime.json

```diff
@@ -76,43 +98,70 @@
       "post": {
         "description": "Gets the predictions for an application version.",
         "operationId": "Prediction_GetVersionPrediction",
+        "consumes": [
```

Comment:
@sergey-shandar I want to confirm this line. It add a new field `consumes` into the method which seem like bring a constraint. Is this a breaking change or not?

Reply:
According to the "BreakingChange" check it's not. This was missing from the original swagger file and nothing new was actually introduced to the service. We've been supporting `applications/json` only from day one. 

I tried to generate the C# SDK from the new file and checked the JS and Go SDKs and none of them had any breaking changes.